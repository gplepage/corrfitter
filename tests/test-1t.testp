(dp0
S'etas:ao'
p1
cnumpy.core.multiarray
_reconstruct
p2
(cnumpy
ndarray
p3
(I0
tp4
S'b'
p5
tp6
Rp7
(I1
(I3
tp8
cnumpy
dtype
p9
(S'f8'
p10
I0
I1
tp11
Rp12
(I3
S'<'
p13
NNNI-1
I-1
I0
tp14
bI00
S'\xac\xb1sk~}\xb1?\xaaFZlk\xdd\xa8\xbf\xb5B\xa3\xa2\xdd\xf3\xc0?'
p15
tp16
bsS'Vno'
p17
g2
(g3
(I0
tp18
g5
tp19
Rp20
(I1
(I3
I3
tp21
g12
I00
S"$\xfd\xa9/\x1c3\xc4\xbf\xef\x16e\xf1U\xf4\xb9\xbf\x9b\x06\xf0\xcf\xf48\xa3?\x1a\x9fb\x1e\xcb\x01\x9f\xbf\xeae=\xca^\xa4W\xbf\xd8v'Jjn\xb3\xbf(Q\xee\xd7\x0f\x9f\xb5\xbf\xd8\xb3E\x8b\xcd\xe2\xa3\xbf\x8c9\xb2\x0c\xd5\xf5\x85?"
p22
tp23
bsS'Vnn'
p24
g2
(g3
(I0
tp25
g5
tp26
Rp27
(I1
(I3
I3
tp28
g12
I00
S'WO_D\xe9N\xc4?\xf6\x88\xde\x008~\xb9\xbf\x0c\xb0\xa1e1?p\xbf\xd1\xcf%+gt\x83?\x02\n7\xc8\xbeq\xd8\xbf\x87\xf3\xa3\xcf\xad\xfc\x99?\xe90j]ElM\xbfk\xdc7\xa8\xa1\xf5\x8d?}d95,\t\x85?'
p29
tp30
bsS'Von'
p31
g2
(g3
(I0
tp32
g5
tp33
Rp34
(I1
(I3
I3
tp35
g12
I00
S'\x1c\x9d\x18\xe0\xbb\xa3\xc4\xbfQ\x99\xeb\xeb\x06.\xb7\xbf\xe7%h>I\xae\xa1?\xb7\xb5H\xbf\xaa\xf3\xb0?\xd2d\xc9\x9dxc\xd0?\xad\t\xa4\x8d\x81\xd4\xa6\xbf\x8d\x83\x9dk\x01\xe7\xa3\xbf\xc6\xcfA\xd7\xf5t\xcd\xbf\xa1\x80\xbf]ee~?'
p36
tp37
bsS'Voo'
p38
g2
(g3
(I0
tp39
g5
tp40
Rp41
(I1
(I3
I3
tp42
g12
I00
S"\x05\xb4\x07\xb8/I\x93\xbf\xac\x86\xbb\xb0\x1ae\xd3\xbf\x15\x96 {\xf0\xce\xd3?\x1e\xd4\x81%\x06\x84\xe2?K\xb5!\xedO/\xc1?\xeb\x97\xb6V\xaa\x03\x80?\xf6\xf6\x90\xfdJ\x97\xdc?\xfbb\xd6O\x8b\n\xac?\x80\xe1\xb8S'\xf3\x82?"
p43
tp44
bsS'Ds:a'
p45
g2
(g3
(I0
tp46
g5
tp47
Rp48
(I1
(I3
tp49
g12
I00
S'\xc19\x05\xde\x11\xfa\xca?\xda\xfa3\xaczh\xcc?\xc2\x9b`\xe1[h\xe0?'
p50
tp51
bsS'log(Ds:dEo)'
p52
g2
(g3
(I0
tp53
g5
tp54
Rp55
(I1
(I3
tp56
g12
I00
S'w\x10\xb9\x1a\xc8g\xd4?@\x03\xdbm\xcd\xd5\xfc\xbfC\x81,\xbc\x86U\xf1\xbf'
p57
tp58
bsS'log(etas:dE)'
p59
g2
(g3
(I0
tp60
g5
tp61
Rp62
(I1
(I3
tp63
g12
I00
S'\x94\x94\x18\x82\xd1m\xd9\xbf\x9f1+\x1f\xcb\xc4\xdf\xbf\x8e\xeeV!\xf9\x89\xb1\xbf'
p64
tp65
bsS'Ds:ao'
p66
g2
(g3
(I0
tp67
g5
tp68
Rp69
(I1
(I3
tp70
g12
I00
S'\x03h\xc0\xb2^\xdf\xa6?\x84 \xa7s\xa8d\xb6?(\xdf\xb1\xd5\xa3\xd0\xbd?'
p71
tp72
bsS'log(Ds:dE)'
p73
g2
(g3
(I0
tp74
g5
tp75
Rp76
(I1
(I3
tp77
g12
I00
S'P\xda\xdd\xd0#X\xd1?\x8c\x1e)g$\xb9\xeb\xbf\x9atQBQ\xe0\xe8\xbf'
p78
tp79
bsS'log(etas:dEo)'
p80
g2
(g3
(I0
tp81
g5
tp82
Rp83
(I1
(I3
tp84
g12
I00
S'\xec\xee\xcbD\xd6\x04\xc4\xbf\xc6\x90\xe6\xe8\xd6a\xe2\xbf\xfa\xfa\x08\x86\xc5\xb1\xd9\xbf'
p85
tp86
bsS'etas:a'
p87
g2
(g3
(I0
tp88
g5
tp89
Rp90
(I1
(I3
tp91
g12
I00
S'\xce\xaf\xb09\x8a/\xc6?\xda\xce,\xe2\x10\x95\xd1?\xe9I\xf9\xf0\x8c\x1e\xe4?'
p92
tp93
bs.