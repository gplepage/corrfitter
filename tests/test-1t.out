========================== nexp = 3
Least Square Fit:
  chi2/dof [dof] = 0.72 [112]    Q = 0.99    logGBF = -271.7    itns = 2

Parameters:
           Ds:a_  0.211 +-   0.001           (    0.01 +-        1)
               _  0.222 +-   0.023           (    0.01 +-        1)
               _  0.513 +-   0.006           (    0.01 +-        1)
     log(Ds:dE)_  0.271 +-   0.000           (    0.26 +-     0.15)
               _ -0.866 +-   0.077           (   -0.69 +-        1)
               _ -0.777 +-   0.027           (   -0.69 +-        1)
          Ds:ao_  0.045 +-   0.009           (    0.01 +-        1)
               _  0.087 +-   0.013           (    0.01 +-        1)
               _  0.116 +-   0.007           (    0.01 +-        1)
    log(Ds:dEo)_  0.319 +-   0.012           (    0.34 +-     0.14)
               _ -1.802 +-   0.253           (   -0.69 +-        1)
               _ -1.083 +-   0.145           (   -0.69 +-        1)
         etas:a_  0.173 +-   0.001           (    0.01 +-        1)
               _  0.275 +-   0.042           (    0.01 +-        1)
               _  0.629 +-   0.127           (    0.01 +-        1)
   log(etas:dE)_ -0.397 +-   0.002           (    -0.4 +-      0.3)
               _ -0.496 +-   0.099           (   -0.69 +-        1)
               _ -0.069 +-   0.250           (   -0.69 +-        1)
        etas:ao_  0.068 +-   0.002           (    0.01 +-        1)
               _ -0.049 +-   0.062           (    0.01 +-        1)
               _  0.132 +-   0.144           (    0.01 +-        1)
  log(etas:dEo)_ -0.156 +-   0.008           (   -0.13 +-     0.23)
               _ -0.574 +-   0.759           (   -0.69 +-        1)
               _ -0.401 +-   0.915           (   -0.69 +-        1)
            Vno_ -0.158 +-   0.006           (    0.01 +-        1)
               _ -0.101 +-   0.014           (    0.01 +-        1)
               _  0.038 +-   0.034           (    0.01 +-        1)
               _ -0.030 +-   0.096           (    0.01 +-        1)
               _ -0.001 +-   0.601           (    0.01 +-        1)
               _ -0.076 +-   0.999           (    0.01 +-        1)
               _ -0.084 +-   0.142           (    0.01 +-        1)
               _ -0.039 +-   0.982           (    0.01 +-        1)
               _  0.011 +-   1.000           (    0.01 +-        1)
            Vnn_  0.159 +-   0.001           (    0.01 +-        1)
               _ -0.100 +-   0.011           (    0.01 +-        1)
               _ -0.004 +-   0.016           (    0.01 +-        1)
               _  0.009 +-   0.008           (    0.01 +-        1)
               _ -0.382 +-   0.983           (    0.01 +-        1)
               _  0.025 +-   1.000           (    0.01 +-        1)
               _ -0.001 +-   0.031           (    0.01 +-        1)
               _  0.015 +-   0.999           (    0.01 +-        1)
               _  0.010 +-   1.000           (    0.01 +-        1)
            Voo_ -0.019 +-   0.104           (    0.01 +-        1)
               _ -0.303 +-   0.207           (    0.01 +-        1)
               _  0.310 +-   0.305           (    0.01 +-        1)
               _  0.579 +-   0.736           (    0.01 +-        1)
               _  0.134 +-   0.990           (    0.01 +-        1)
               _  0.008 +-   1.000           (    0.01 +-        1)
               _  0.447 +-   0.824           (    0.01 +-        1)
               _  0.055 +-   0.995           (    0.01 +-        1)
               _  0.009 +-   1.000           (    0.01 +-        1)
            Von_ -0.161 +-   0.010           (    0.01 +-        1)
               _ -0.091 +-   0.019           (    0.01 +-        1)
               _  0.035 +-   0.038           (    0.01 +-        1)
               _  0.066 +-   0.097           (    0.01 +-        1)
               _  0.256 +-   0.620           (    0.01 +-        1)
               _ -0.045 +-   0.999           (    0.01 +-        1)
               _ -0.039 +-   0.183           (    0.01 +-        1)
               _ -0.230 +-   0.936           (    0.01 +-        1)
               _  0.007 +-   1.000           (    0.01 +-        1)

etas:dE = 0.672(1)  0.61(6)  0.9(2)
 etas:E = 0.672(1)  1.28(6)  2.2(3)
 etas:a = 0.173(1)  0.27(4)  0.6(1)

Ds:dE = 1.3113(5)  0.42(3)  0.46(1)
 Ds:E = 1.3113(5)  1.73(3)  2.19(2)
 Ds:a = 0.2108(8)  0.22(2)  0.513(6)

etas->V->Ds  = 0.1587(9)
etas->V->Dso = -0.158(6)

Partial % Errors:
                           Vno       EDs     Eetas       Vnn
------------------------------------------------------------
            etas:ao:     0.111     0.001     0.008     0.003
      log(etas:dEo):     0.470     0.003     0.017     0.014
                Vno:     0.618     0.002     0.012     0.031
                Vnn:     0.049     0.003     0.011     0.310
                Von:     1.247     0.001     0.013     0.060
                Voo:     0.763     0.003     0.021     0.056
               Ds:a:     0.008     0.001     0.001     0.004
        log(Ds:dEo):     0.219     0.003     0.002     0.023
       log(etas:dE):     0.262     0.003     0.026     0.006
              Ds:ao:     0.017     0.000     0.000     0.001
         log(Ds:dE):     0.039     0.003     0.004     0.014
              stat.:     3.397     0.040     0.155     0.483
             etas:a:     0.126     0.001     0.012     0.003
------------------------------------------------------------
              total:     3.798     0.041     0.162     0.582




